tests/generator-Tests/expected.ji/Core_Jar2Xml/Android.Text.ISpanned.cs

```diff
@@ -5,14 +5,34 @@
 
 namespace Android.Text {
 
+	[Register ("android/text/Spanned", DoNotGenerateAcw=true)]
+	public abstract class Spanned : Java.Lang.Object {
+
+		internal Spanned ()
+		{
+		}
+
+		// Metadata.xml XPath field reference: path="/api/package[@name='android.text']/interface[@name='Spanned']/field[@name='SPAN_COMPOSING']"
+		[Register ("SPAN_COMPOSING")]
+		public const int SpanComposing = (int) 256;
+	}
+
+	[Register ("android/text/Spanned", DoNotGenerateAcw=true)]
+	[global::System.Obsolete ("Use the 'Spanned' type. This type will be removed in a future release.", error: true)]
+	public abstract class SpannedConsts : Spanned {
+
+		private SpannedConsts ()
+		{
+		}
+	}
+
 	// Metadata.xml XPath interface reference: path="/api/package[@name='android.text']/interface[@name='Spanned']"
 	[Register ("android/text/Spanned", "", "Android.Text.ISpannedInvoker")]
 	public partial interface ISpanned : IJavaObject, IJavaPeerable {
 
 		// Metadata.xml XPath method reference: path="/api/package[@name='android.text']/interface[@name='Spanned']/method[@name='getSpanFlags' and count(parameter)=1 and parameter[1][@type='java.lang.Object']]"
-		[return:global::Android.Runtime.GeneratedEnum]
-		[Register ("getSpanFlags", "(Ljava/lang/Object;)I", "GetGetSpanFlags_Ljava_lang_Object_Handler:Android.Text.ISpannedInvoker, ")]
-		Android.Text.SpanTypes GetSpanFlags (Java.Lang.Object tag);
```

Comment:
The "loss" of enum types here & elsewhere is *weird*.

Reply:
I accidentally removed the `fields.xml` and `methods.xml` transforms. Added them back.